hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/ReadableTokenRelationStoreImpl.java

```diff
@@ -49,9 +51,9 @@ public ReadableTokenRelationStoreImpl(@NonNull final ReadableStates states) {
      * {@inheritDoc}
      */
     @Override
-    public Optional<TokenRelation> get(final long accountNum, final long tokenNum) {
-        final var tokenRelation =
-                Objects.requireNonNull(readableTokenRelState).get(EntityNumPair.fromLongs(accountNum, tokenNum));
+    public Optional<TokenRelation> get(@NonNull final AccountID accountId, @NonNull final TokenID tokenId) {
+        final var tokenRelation = Objects.requireNonNull(readableTokenRelState)
+                .get(EntityNumPair.fromLongs(accountId.accountNumOrThrow(), tokenId.tokenNum()));
```

Comment:
accountId.accountNumOrElse(AccountID.DEFAULT) . Similarly for token ?

Reply:
Why wouldn't we want to fail as soon as possible?

Reply:
The answer is because we're not sure which status is returned when an NPE is thrown. 

I fixed it